mlrun/api/runtime_handlers/__init__.py

```diff
@@ -49,6 +52,7 @@ def get_runtime_handler(kind: str) -> BaseRuntimeHandler:
         RuntimeKinds.spark: SparkRuntimeHandler,
         RuntimeKinds.remotespark: RemoteSparkRuntimeHandler,
         RuntimeKinds.job: KubeRuntimeHandler,
+        RuntimeKinds.databricksruntime: DatabricksRuntimeHandler,
```

Comment:
```suggestion
        RuntimeKinds.databricks: DatabricksRuntimeHandler,
```
Forgot to change this.